utils/data_loader_fv3gfs.py

```diff
@@ -97,23 +97,26 @@ def _get_files_stats(self):
         logging.info(f"Following variables are available: {list(self.ds.variables)}.")
 
     def _load_stats_data(self):
-        logging.info(f"Opening mean stats data at {self.params.global_means_path}")
-        means_ds = netCDF4.Dataset(self.params.global_means_path)
-        means_ds.set_auto_mask(False)
-        self.in_means = np.array([means_ds.variables[c][:] for c in self.in_names])
-        self.out_means = np.array([means_ds.variables[c][:] for c in self.out_names])
-        self.in_means = self.in_means.reshape((1, self.n_in_channels, 1, 1))
-        self.out_means = self.out_means.reshape((1, self.n_out_channels, 1, 1))
-        means_ds.close()
+        logging.info(f"Opening global mean stats data at {self.params.global_means_path}")
```

Comment:
Not so sure how helpful this will be if we are mounting datasets within docker images, but can't hurt!

Reply:
Agreed.